internal/server/models/pagination.go

```diff
@@ -0,0 +1,70 @@
+package models
+
+import "github.com/infrahq/infra/api"
+
+// Internal Pagination Data
+type Pagination struct {
+	Page    int
+	Limit   int
+	Sort    string
+	MaxPage int
+	Next    int
+	Prev    int
+}
+
+// function to convert API --> this object
+// method to convert Pagination to response
```

Comment:
There's a standard way to comment functions in go: https://go.dev/doc/effective_go#commentary

Pretty much you start the comment with the name of the function, for example:
```
// RequestToPagination converts an API request object to an internal pagination model
```

It's kinda redundant, but makes for consistency.